CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Unreleased
 
+### General
+
+- Added public api constants for the roles of `wl_shell_surface`, `zxdg_toplevel` and `xdg_toplevel`. See the
+  `shell::legacy` and `shell::xdg` modules for these constants.
+- Whether a surface is toplevel equivalent can be determined with the new function `shell::toplevel_like`. 
```

Comment:
Looks like this line needs to be updated to the new name you set in your last commit.